cfy_cluster_setup/main.py

```diff
@@ -341,10 +360,22 @@ def _validate_config(config):
     _check_existing_vms(config)
 
 
-def install(config_path):
+def generate_config(output_path, verbose):
+    setup_console_logger(verbose)
+    output_path = output_path or CLUSTER_INSTALL_CONFIG_PATH
+    if isdir(output_path):
+        output_path = join(output_path, CLUSTER_CONFIG_FILE_NAME)
+    copy(join(dirname(__file__), CLUSTER_CONFIG_FILE_NAME), output_path)
+    logger.info('Created the cluster install configuration file in %s',
+                output_path)
+
+
+def install(config_path, verbose):
+    setup_console_logger(verbose)
     if not yum_is_present():
         raise ClusterInstallError('Yum is not present.')
 
+    logger.info('Installing an Active-Active Cloudify cluster')
```

Comment:
Maybe remove the Active-Active reference since we only offer one type of cluster? Don't want the support team to have to field questions about how to make it active-passive.

Reply:
Right, I'll remove it.